app/helpers/action_links_helper.rb

```diff
@@ -18,7 +21,7 @@ def renew_link_for(resource)
   end
 
   def display_details_link_for?(resource)
-    a_transient_registration?(resource) || a_registration?(resource)
```

Comment:
we should probably get rid of this method. Whatever the search result returns, will have a detail link now :)

Reply:
Agreed. It's a goner!